static/js/tools/timeline/page.test.tsx

```diff
@@ -15,7 +15,6 @@
  */
 
 jest.mock("axios");
-// jest.mock("../../chart/draw");
```

Comment:
rm

Reply:
To clarify, this change is to remove the commented out line (I forgot to remove it before merging https://github.com/datacommonsorg/website/pull/3158) since `../../chart/draw` no longer exists.

Should I still revert this change?

Reply:
yes sorry just to remove the comment